python/src/trezorlib/debuglink.py

```diff
@@ -245,19 +332,30 @@ def state(self) -> messages.DebugLinkState:
         return self._call(messages.DebugLinkGetState())
 
     def read_layout(self) -> LayoutContent:
-        return LayoutContent(self.state().layout_lines)
+        return LayoutContent(self.state().json_content or "")
+
+    def wait_layout(self, wait_for_external_change: bool = False) -> LayoutContent:
+        # Next layout change will be caused by external event
```

Comment:
This is rather a synchronization mechanism - when there is a change that we did not do directly by debug events - device being locked, running device_handler.run() etc., it will wait for this external change

Reply:
no, what I meant is, why would you ever call `wait_layout(external_event=False)`. it kind of seems to me that, given the availability of `wait=True` on debuglink decisions, we could declare `wait_layout()` to mean "wait for a change caused externally" in all cases.

could that work or are there uses that are not covered?